cmd/backup-manager/app/backup/manager.go

```diff
@@ -114,42 +114,65 @@ func (bm *BackupManager) performBackup(backup *v1alpha1.Backup, db *sql.DB) erro
 	}
 	glog.Infof("cluster %s %s is %s", bm, constants.TikvGCVariable, oldTikvGCTime)
 
-	err = bm.setTikvGCLifeTime(db, constants.TikvGCLifeTime)
+	oldTikvGCTimeDuration, err := time.ParseDuration(oldTikvGCTime)
 	if err != nil {
-		glog.Errorf("cluster %s set tikv GC life time to %s failed, err: %s", bm, constants.TikvGCLifeTime, err)
+		glog.Errorf("cluster %s parse old %s failed, err: %s", bm, constants.TikvGCVariable, err)
 		return bm.StatusUpdater.Update(backup, &v1alpha1.BackupCondition{
 			Type:    v1alpha1.BackupFailed,
 			Status:  corev1.ConditionTrue,
-			Reason:  "SetTikvGCLifeTimeFailed",
+			Reason:  "ParseOldTikvGCLifeTimeFailed",
 			Message: err.Error(),
 		})
 	}
-	glog.Infof("set cluster %s %s to %s success", bm, constants.TikvGCVariable, constants.TikvGCLifeTime)
-
-	backupFullPath, err := bm.dumpTidbClusterData()
+	tikvGCTimeDuration, err := time.ParseDuration(constants.TikvGCLifeTime)
```

Comment:
What about defining the default gc lifetime as `time.Duration` so we don't have to parse this every time.

Reply:
a good idea.I'll try to change it later.